build/source/engine/coupled_em.f90

```diff
@@ -132,15 +131,11 @@ subroutine coupled_em(&
  ! the model solver
  USE indexState_module,only:indexState      ! define indices for all model state variables and layers
  USE opSplittin_module,only:opSplittin      ! solve the system of thermodynamic and hydrology equations for a given substep
+ USE time_utils_module,only:elapsedSec      ! calculate the elapsed time
```

Comment:
Is this needed given that the changes use built-in `CPU_TIME(out_var)` which already returns the timing info in `[s]`?